src/Models/GameModel.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace GameCollection\Models;
+
+use PDO;
+use GameCollection\Entities\Game;
+
+class GameModel
+{
+    private PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+    public function getAllGames()
+    {
+        $query = $this->db->prepare("SELECT *
```

Comment:
You should ditch the \* and instead be more specific with which fields you're selecting. At the moment you're selecting fields that never actually get used